docs/docs/sourcing-from-prismic.md

```diff
@@ -68,160 +60,142 @@ require("dotenv").config({
 
-Single pages (like your homepage, privacy policy page etc.) don't need [GraphQL connections](https://blog.apollographql.com/explaining-graphql-connections-c48b7c3d6976) (e.g. `allPrismicHomepage`) due to the fact that only one document for that type exists in Prismic anyway. Therefore you need to use `prismicHomepage` for your query. This also has the benefit that you don't have to map over an array. Your page could look something like this:
-
-```jsx:title=src/pages/index.js
-import React from "react"
-import { graphql, Link } from "gatsby"
-
-const Index = ({ data: { prismicHomepage } }) => (
-  <React.Fragment>
-    <h1>{prismicHomepage.data.title.text}</h1>
-    <div
-      dangerouslySetInnerHTML={{ __html: prismicHomepage.data.content.html }}
-    />
-  </React.Fragment>
-)
-
-export default Index
-
-export const pageQuery = graphql`
-  query IndexQuery {
-    prismicHomepage {
-      data {
-        title {
-          text
-        }
-        content {
-          html
-        }
-      }
-    }
-  }
-`
-```
 
 ## Wrapping up
 
-This was an example meant to help you understand how Prismic works with Gatsby. With your newfound knowledge of Prismic (and perhaps even Gatsby), you're now able to:
-
-- Create a Prismic repository and setting it up together with the Gatsby plugin
-- Query data from Prismic and using it to programmatically create blogpost pages
-- Use Prismic together with Netlify
-- Add relationships between posts, e.g. with categories
-- Query data from Prismic for single pages
+This was an example meant to help you understand how Prismic works with Gatsby. The videos in this article where provided by [LekoArts](https://www.youtube.com/channel/UCbxFBnrkEMExO_QX8tkJ76Q)  
 
-As mentioned in the beginning of this guide, if you got stuck, you can compare your code to the [gatsby-starter-prismic](https://github.com/LeKoArts/gatsby-starter-prismic) which is the project set up in the videos. A working example created by following this guide is available in the [commit history](https://github.com/LeKoArts/gatsby-starter-prismic/tree/4aa5d52e79a0b4d90f0a671c24eb8289eb15a42b) of the aforementioned starter. More advanced usages of Prismic in Gatsby would be [Slices](https://intercom.help/prismicio/content-modeling-and-custom-types/field-reference/slices) and [Labels](https://intercom.help/prismicio/content-modeling-and-custom-types/structure-your-content/add-custom-styles-to-rich-text).
+With your newfound knowledge of Prismic (and perhaps even Gatsby), you're now able to:
 
-<!-- Links to more advanced tutorials will go here -->
+- Create a Prismic repository and setting it up together with the Gatsby plugin
```

Comment:
This kind of wrap-up is more typical of a tutorial than a reference guide. Rather than including recap steps like this, the headings of each section can provide an overview. Following the Reference Guide template, a better option would be to include a list of additional resources (perhaps the ones from the top of the guide, since that's a-standard of a guide as well?): https://www.gatsbyjs.org/contributing/how-to-write-a-reference-guide#reference-guide-template

Reply:
Ok, I've modified this to be a list